Web/View/LandingPages/Show.hs

```diff
@@ -84,4 +92,24 @@ renderParagraphQuote paragraphQuote =
         , subtitle = paragraphQuote.subtitle
         , imageUrl = paragraphQuote.imageUrl |> fromMaybe ""
         }
-        |> Web.Element.Quote.render
\ No newline at end of file
+        |> Web.Element.Quote.render
+
+renderParagraphHeroImage :: ParagraphHeroImage -> Html
+renderParagraphHeroImage paragraphHeroImage =
+    let
+        button = case paragraphHeroImage.link of
+            Just linkText ->
+                Just RenderButton
+                    { text = "Read More"
+                    , url = linkText
+                    , isPrimary = True
+                    }
+            _ -> Nothing
```

Comment:
with using fmap - if value is Nothing, it won't map over the value -- and will remain as `Nothing`

```suggestion
        maybeButton = paragraphHeroImage.link |> fmap (\linkText -> ...)
```